app/src/components/widgets/charts/investigatedTrendChart/investigatedTrendChart.jsx

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright 2017 EPAM Systems
+ *
+ *
+ * This file is part of EPAM Report Portal.
+ * https://github.com/reportportal/service-ui
+ *
+ * Report Portal is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Report Portal is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Report Portal.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { intlShape, injectIntl } from 'react-intl';
+import { Component } from 'react';
+import 'c3/c3.css';
```

Comment:
Not needed. This file is already included at the top level.

Reply:
removed